packages/design-system/image/image.tsx

```diff
@@ -30,10 +32,15 @@ function StyledImage({
 }: ImageProps) {
   return (
     <StyledExpoImage
-      source={source}
       style={[{ height, width, borderRadius }, style as any]}
       contentFit={contentFit ?? resizeMode}
       placeholder={{ blurhash, width, height }}
+      source={{
+        ...(typeof source === "object" && source),
```

Comment:
Are there any places where we pass a `require` instead a source? If yes, this might break, even though its a copy from my App

Reply:
yeah, good catch!